channels/ru.m3u

```diff
@@ -502,18 +412,12 @@ http://hls-rossp.cdnvideo.ru/hls/5ace67f0dc96bf0a90d8a2b7/playlist.m3u8
 http://hls-rossp.cdnvideo.ru/hls/5ace6814e563f22e4ddc1f44/playlist.m3u8
 #EXTINF:-1 tvg-id="LitKlubTVStihi.ru" tvg-name="ЛитКлуб ТВ - Стихи" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",ЛитКлуб ТВ - Стихи
 http://hls-rossp.cdnvideo.ru/hls/5aac88ffdc96bf05a7f78899/playlist.m3u8
-#EXTINF:-1 tvg-id="LitKlubTVProza.ru" tvg-name="ЛитКлуб ТВ Проза" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",ЛитКлуб ТВ Проза (720p)
-http://video.platformcraft.ru/hls/rossp/proza720/playlist.m3u8
```

Comment:
works

Reply:
Redirects to the Rossiya link above.

Reply:
Which one? Those three channels are all about different kinds of poetry and I got some differnt streams

Reply:
All links that were removed redirects to their Rossiya variant

(for example, if you use Firefox, and use the Network sniffer, you'll see a redirection going to the hls link above)

Reply:
Would you be kind to show me the final link, please? I don't see any other LitClubs in the playlist

Reply:
![image](https://user-images.githubusercontent.com/30985701/128013141-29989e82-6fb1-4a27-adb9-72d04c2ca778.png)

Reply:
Ah yes, my bad. Tho I'd keep both links in case those hashes gonna change over time

Reply:
That network sniffer is really powerful. I see you got that extension to work. Really makes it convenient. Another extension I use is Clear URL. It strips all those requests about your browser before sending to server. Most of the URL now-a-days are spying using the URL. That's how they know you are not using their app, but instead using a browser.

Reply:
@Dum4G Well, I don't know if it's useful to keep the other link if it just redirects to the source link we also have. Better change those links directly and making sure we have no redirections whatsoever.